buildconfig/Jenkins/buildscript

```diff
@@ -95,7 +97,7 @@ git clean -d -x --force --exclude=${BUILD_DIR_REL} --exclude=".Xauthority-*"
 # Print out the versions of things we are using
 ###############################################################################
 # we use cmake3 on rhel because cmake is too old
-if [ $(command -v cmake3) ]; then
+if [ $(command -v cmake3) ] && [[ ! $ON_CONDA ]]; then
```

Comment:
Just out of interest, is the `cmake3` check not enough?

Reply:
cmake3 doesn't exist inside of Conda if that makes sense? So when we call cmake inside of conda on RHEL/CentOS7 it gives us version 3, but we can't call cmake3 because there is no file to link them together, without editing the environment we need to call JUST cmake.

Reply:
Oh yes I see now. It's if we are on CentOS, cmake3 is installed but we want to do a conda build. 👍🏻